src/classes/BDI_Donations.cls

```diff
@@ -416,8 +416,22 @@ public class BDI_Donations {
             di.Donation_Stage__c = defaultClosedWonStage;
         }
 
+        Boolean useCustomMappingInsteadOfCurrencyIsoCode = false;
+
+        // Set useCustomMappingInsteadOfCurrencyIsoCode
+        for (string strDIField : mapDIFieldToOppField.keySet()) {
+            String currentValue = mapDIFieldToOppField.get(strDIField);
+
+            // If we have a custom mapping against the field CurrencyIsoCode of the Data Import object.
+            if (strDIField != 'CurrencyIsoCode' && currentValue == 'CurrencyIsoCode') {
+                    useCustomMappingInsteadOfCurrencyIsoCode = true;
+                    continue;
```

Comment:
Think we want a break statement here, not a continue.  Once we set the variable to true, no need to keep looping, correct?

Reply:
correct.